examples/laptop.yml

```diff
@@ -45,10 +47,8 @@ inputs:
 
 # if supported by the launcher, restart the pipeline where it left off
 # if interrupted
-resume: True
```

Comment:
can you change this back to the default also.

Reply:
Done.